os/cluster-architectures.md

```diff
@@ -60,22 +49,44 @@ coreos:
+
+```sh
+$ export DOCKER_HOST=tcp://localhost:2375
+$ docker ps
+```
+
+This avoids discrepencies between your development and production environemnts.
+
+### Related local installation tools
+
+There are several different options for testing Container Linux locally:
+
+- [coreos-vagrant][coreos-vagrant] is not an officially fully supported platform, however may be a good resource to check out if you are already comfortable with Vagrant and are unable to use one of Ignition's officially supported platforms.
+- [Minikube][minikube] is used for local Kubernetes development. This does not use CoreOS but is very fast to setup and is the easiest way to test-drive use Kubernetes.
+
 ## Small cluster
 
 <img class="img-center" src="img/small.png" alt="Small Container Linux Cluster Diagram"/>
 <div class="caption">Small Container Linux cluster running etcd on all machines</div>
 
-| Cost | Great For | Set Up Time | Production |
-|------|-----------|-------------|------------|
-| Low | Small clusters, trying out Container Linux | Minutes | Yes |
+| Cost | Great For                                  | Set Up Time | Production |
+|------|--------------------------------------------|-------------|------------|
+| Low  | Small clusters, trying out Container Linux | Minutes     | Yes        |
 
-For small clusters between 3-9 machines, running etcd on all of the machines allows for high availability without paying for extra machines that just run etcd.
+For small clusters, between 3-9 machines, running etcd on all of the machines allows for high availability without paying for extra machines that just run etcd.
 
-Getting started is easy &mdash; a single cloud-config can be used to start all machines on a cloud-provider.
+Getting started is easy &mdash; a single Container Linux config can be used to provision all machines on your cloud-provider.
 
 ### Configuring the machines
 
-Following the guide for each of the [supported platforms](https://coreos.com/docs#running-coreos) will be the easiest way to get started with this architecture. Boot the desired number of machines with the same cloud-config and discovery token. The cloud-config specifies that etcd and fleet will be started on each machine.
+For more information on getting started with this architecture, see the CoreOS documentation on [supported platforms][coreos-supported]. These include [Amazon EC2][coreos-ec2], [Openstack][coreos-openstack], [Azure][coreos-azure], [Google Compute Platform][coreos-gce], [bare metal iPXE][coreos-bm], [Digital Ocean][coreos-do], and many more community supported platforms.
+
+Boot the desired number of machines with the same Container Linux config and discovery token. The Container Linux config specifies which services will be started on each machine.
 
 ## Easy development/testing cluster
```

Comment:
Ideally we would have a Kubernetes cluster, but we either need to (A) decide to leave that as an exercise for the reader or (B) figure out a concise way to get K8s up and running on these machines (or (C) bring fleet back for just this tutorial. Juuuust kidding.)